src/pages/trade/[pair].tsx

```diff
@@ -55,10 +63,12 @@ export default function Page() {
     }
   }
 
+  // TOOD eur is not supported through token type
+  // TODO figure out how to reconcile usdc and usd pairs
   return (
     <SidebarLayout className="pt-11">
       <div>
-        <TradeSidebar />
+        <TradeSidebar inputPayToken={token} outputPayToken={currency} />
```

Comment:
looking at the way GMX works, the input pay token  _isn't_ the same as the what shows up on the chart. The chart should reflect the token of the position you want to enter, ie Long SOL, Short SOL, etc, but what you pay can be independently selected with respect to the price chart. On GMX, if you select a different pay token, the chart stays the same. In our case, both the chart and the URL should stay the same